Scripts/AutoSPSourceBuilder.ps1

```diff
@@ -278,7 +282,9 @@ If (!([Security.Principal.WindowsPrincipal] [Security.Principal.WindowsIdentity]
 {
     Write-Warning " - You should run this script under an elevated PowerShell prompt. Launch an elevated PowerShell prompt by right-clicking the PowerShell shortcut and selecting `"Run as Administrator`"."
     Write-Warning " - Running without elevation may cause certain things to fail, e.g. file extraction."
-    Pause -action "proceed if you are sure this is OK, or Ctrl-C to exit" -key "y"
+    if ( !$Unattend ) {
```

Comment:
Could you please follow the style convention here and elsewhere (open brace on new line, watch out for extra/trailing spaces, etc.)?